### ISSUE TITLE
Keep the mic device open during a call

### ISSUE THREAD

**Issue by dbkr** [vector-im/element-call#1412]
We currently close both the video and audio devices when the user mutes, opening them again when the user unmutes. This is great for video because is means the webcam light turns off giving the user confidence that nobody can see them, but for audio, mobile browsers tend to kill the page unless we keep a mic device open to convince it that we're on a call (experimentally, iOS currently works with the mic muted but Android unloads the page fairly quickly). Additionally, users expect to be able to speak as soon as they hit the unmute button, but opening an audio device can take time so we could lose the start of what someone is saying, especially if they're using a bluetooth mic which takes longer to open the device.

This will mean that we'll keep any mic indicators on even when muted, but this seems fairly standard behaviour (Jitsi does it) and on mobile is actually somewhat reassuring to know that the call is still there in the background.

tl;dr: Continue releasing video devices on video mute but keep audio devices open even when muted.

**Comment by dbkr** [comment]
Having just retested, the same Android device is now disconnecting from webrtc as soon as I put chrome into the background. The same is now happening with Jitsi too. That plus https://bugs.chromium.org/p/chromium/issues/detail?id=951418 leads me to think we should not expect a call to continue working on Android if the browser is backgrounded.
